.github/dependabot.yml

```diff
@@ -5,7 +5,8 @@ updates:
     open-pull-requests-limit: 20
```

Comment:
Bump this up.
We have in 9.4.x an average of 5.8 dependabot PRs per week.
Lets make 9.4.x have a limit of 30.

Raw numbers from the start of the year.
```
9.4.x - Daily updates
  
  is:pr created:>=2022-01-01 base:jetty-9.4.x 
      4 open
    174 closed
   (7.9 PRs per week)
  is:pr created:>=2022-01-01 base:jetty-9.4.x label:dependencies
    128 closed
   (5.8 PRs per week)
  is:pr created:>=2022-01-01 base:jetty-9.4.x label:dependencies is:merged 
    105 merged
   (4.7 PRs per week)
  is:pr created:>=2022-01-01 base:jetty-9.4.x label:dependencies is:unmerged 
     23 un-merged
   (22% not merged)  
```